tests/Dfe.PlanTech.Application.UnitTests/Dfe.PlanTech.Application.UnitTests.csproj

```diff
@@ -10,6 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Http" Version="2.1.22" />

```

Comment:
This has been deprecated, use

` <FrameworkReference Include="Microsoft.AspNetCore.App" />` instead mate.

Reply:
This doesn't find DefaultHttpContext

Reply:
Should work; is your dotnet SDK up to date?

I've made the change in [this branch](https://github.com/DFE-Digital/plan-technology-for-your-school/tree/test/use-framework-reference) to demonstrate - does this work for you? Results:

<img width="984" alt="image" src="https://github.com/DFE-Digital/plan-technology-for-your-school/assets/131891854/1f45335e-84c7-4e50-82ed-55db93dfaaf0">

Reply:
Strangely, I have it working in the web test project

Reply:
I've commented this one test out,  I;m not going to let it stop my work.  It's only for one test.